app.config.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
     process.env.NEXT_PUBLIC_PRIVACY_PREFERENCE_CENTER || 'true',
 
   // Default terms to be used for service offerings made on this marketplace
-  defaultAccessTerms:
+  defaultTermsAndConditionsUrl:
     'https://raw.githubusercontent.com/deltaDAO/mvg-portal/v4/content/pages/terms.md',
```

Comment:
suggestion(non-blocking): while we are at it, we should replace this with the portal.../terms url instead